carlos-calc/src/App.js

```diff
@@ -0,0 +1,80 @@
+import { Component } from 'react';
+import "./App.css";
+import Keypad from "./Keypad";
+import CalcOutput from "./CalcOutput";
+
+class App extends Component {
+  constructor() {
+    super(); 
+    this.state = {
+      result: "0",
+      operators: "",
+      num1: "",
+      previousNum: ""
+    }
+  } 
+  handleClick = (e) => {
+    const { result } = this.state;
+    this.setState({ result: result + e.target.value });
+  };
+  addition = () => {
+    this.state.previousNum = this.state.result;
+    this.setState({ result: "" });
+    this.state.operators = "+"; 
+  };
+  subtract = () => {
+    this.state.previousNum = this.state.result;
+    this.setState({ result: "" });
+    this.state.operators = "-";
+  };
+
+  division = () => {
+    this.state.previousNum = this.state.result;
+    this.setState({ result: "" });
+    this.state.operators = "/";
+  };
+  multiplication = () => {
+    this.state.previousNum = this.state.result;
+    this.setState({ result: "" });
+    this.state.operators = "*";
+  };
+
+
+calculate = () => { 
+  this.state.num1 = this.state.result
```

Comment:
destructoring operators out of state makes for less repetition. 
const { operators} = this.state; 
A switch statement would be better suited for this conditional. 
switch(operation) {
 case: "+":